.gitlab-ci.yml

```diff
@@ -1,5 +1,6 @@
 # ------------------------------------------------------------------------------
 # The testing matrix considers ruby/puppet versions supported by SIMP and PE:
+      if (( os == 'Amazon' ) && ( "#{release}" == '2' ))
```

Comment:
This and breaks the `.gitlab-ci.yml` and looks like it's probably misplaced Ruby copypasta meant for something else.

```suggestion
```